spanner/client.go

```diff
@@ -867,13 +867,20 @@ func (c *Client) BatchReadOnlyTransaction(ctx context.Context, tb TimestampBound
 		err error
 	)
 
-	// Create session.
-	s, err = c.sc.createSession(ctx)
-	if err != nil {
-		return nil, err
+	if c.idleSessions.isMultiplexedSessionForPartitionedOpsEnabled() {
```

Comment:
Where are we validating that the env for pops and global env for mux is set to true?

Reply:
https://github.com/googleapis/google-cloud-go/pull/11583/files#diff-a7eef968d026e320839e33486c1ce4e449d69d9f6c86052edeae4528b60b5579R505

Reply:
Got it.